internal/provider/resource_rediscloud_subscription_peering.go

```diff
@@ -229,7 +233,15 @@ func resourceRedisCloudSubscriptionPeeringRead(ctx context.Context, d *schema.Re
 		return diag.FromErr(err)
 	}
 
-	providerName := d.Get("provider_name").(string)
+	providerName := "AWS"
+
+	if redis.StringValue(peering.GCPProjectUID) != "" {
+		providerName = "GCP"
```

Comment:
Is there a case where the `GCPProjectUID` could get removed but it still be a GCP peering?